drivers/ovsdbDriver.go

```diff
@@ -151,7 +151,18 @@ func (d *OvsdbDriver) Echo([]interface{}) {
 }
 
 func (d *OvsdbDriver) performOvsdbOps(ops []libovsdb.Operation) error {
-	reply, _ := d.ovs.Transact(ovsDataBase, ops...)
+	log.Println(ops)
+	var reply []libovsdb.OperationResult
+	var op libovsdb.Operation
+	if len(ops) == 1 {
```

Comment:
I'm not sure if you still need this change..
Please see if it works without this change. 
Either way, log messages needs to be cleaned up..